### PR TITLE
Add support for updating ARM tags on AKS clusters

### PR DESCRIPTION
**Pull request by seanmck**
This PR adds support for updating RM tags on AKS clusters.

---
- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

### CODE REVIEW

**Comment by haroldrandom** [comment]
Are this PR still active?

**Comment by tjprescott** [comment]
@haroldrandom it doesn't seem so. I will dismiss my stale review.

**Comment by haroldrandom** [comment]
There are some conflicts, could you please solve them first?
I don't know the context.

**Comment by tjprescott** [comment]
@seanmck

**Comment by yungezz** [comment]
hi @seanmck  are you still on this PR? if yes, could you pls sync with master branch, and add test? thanks.

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -679,9 +679,10 @@ def aks_update(cmd, client, resource_group_name, name, enable_cluster_autoscaler
...
                api_server_authorized_ip_ranges=None,
                enable_pod_security_policy=False,
-               disable_pod_security_policy=False):
+               disable_pod_security_policy=False,
+               tags=None):
     update_flags = enable_cluster_autoscaler + disable_cluster_autoscaler + update_cluster_autoscaler
-    if update_flags != 1 and api_server_authorized_ip_ranges is None and \
+    if update_flags != 1 and api_server_authorized_ip_ranges is None and tags is None and \
        (enable_pod_security_policy is False and disable_pod_security_policy is False):
         raise CLIError('Please specify "--enable-cluster-autoscaler" or '
```
Comment (arrownj) [inline src/aks-preview/azext_aks_preview/custom.py:687]: Add `--tags` here ?

**Comment by yungezz** [comment]
@seanmck  thanks for contribution, close the PR for now since no activity. pls feel free to create new PR when ready, thanks.

**Pull request closed without merging**